server/GraknServer.java

```diff
@@ -26,70 +26,66 @@
 import grakn.core.common.parameters.Options;
 import grakn.core.concurrent.common.Executors;
 import grakn.core.migrator.MigratorClient;
-import grakn.core.rocks.RocksGrakn;
+import grakn.core.rocks.Factory;
+import grakn.core.rocks.RocksFactory;
 import grakn.core.server.common.ServerCommand;
 import grakn.core.server.common.ServerDefaults;
+import grakn.core.server.common.Util;
 import io.grpc.Server;
 import io.grpc.netty.NettyServerBuilder;
 import io.netty.channel.socket.nio.NioServerSocketChannel;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import picocli.CommandLine;
-import picocli.CommandLine.ParameterException;
-import picocli.CommandLine.PropertiesDefaultProvider;
-import picocli.CommandLine.UnmatchedArgumentException;
 
-import java.io.FileInputStream;
 import java.io.IOException;
 import java.net.BindException;
-import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.time.Duration;
 import java.time.Instant;
-import java.util.Map;
-import java.util.Properties;
 import java.util.concurrent.TimeUnit;
 import java.util.logging.Level;
 
 import static grakn.core.common.exception.ErrorMessage.Server.ALREADY_RUNNING;
 import static grakn.core.common.exception.ErrorMessage.Server.DATA_DIRECTORY_NOT_FOUND;
 import static grakn.core.common.exception.ErrorMessage.Server.DATA_DIRECTORY_NOT_WRITABLE;
-import static grakn.core.common.exception.ErrorMessage.Server.ENV_VAR_NOT_FOUND;
 import static grakn.core.common.exception.ErrorMessage.Server.EXITED_WITH_ERROR;
 import static grakn.core.common.exception.ErrorMessage.Server.FAILED_AT_STOPPING;
-import static grakn.core.common.exception.ErrorMessage.Server.FAILED_PARSE_PROPERTIES;
-import static grakn.core.common.exception.ErrorMessage.Server.PROPERTIES_FILE_NOT_FOUND;
+
 import static grakn.core.common.exception.ErrorMessage.Server.UNCAUGHT_EXCEPTION;
-import static grakn.core.server.common.ServerDefaults.ASCII_LOGO_FILE;
-import static grakn.core.server.common.ServerDefaults.PROPERTIES_FILE;
+import static grakn.core.server.common.Util.parseCommandLine;
+import static grakn.core.server.common.Util.parseProperties;
+import static grakn.core.server.common.Util.printASCIILogo;
 
 
 public class GraknServer implements AutoCloseable {
 
     private static final Logger LOG = LoggerFactory.getLogger(GraknServer.class);
 
-    private final Grakn grakn;
-    private final Server server;
-    private final ServerCommand.Start command;
-    private final GraknService graknService;
-    private final MigratorService migratorRPCService;
+    protected final ServerCommand.Start command;
```

Comment:
May want to discuss how to order the fields here. I think `server, `graknService`, and `migratorService` should be grouped together. Additionally, `factory` and `grakn` should also be close together.